clippy_lints/src/module_style.rs

```diff
@@ -33,131 +32,117 @@ declare_clippy_lint! {
     ///   stuff.rs
     ///   lib.rs
     /// ```
-    pub MODULE_STYLE,
-    style,
+    pub SELF_NAMED_MOD_FILE,
```

Comment:
This is flip-flopped. We want `#[warn(self_named_mod_files)]` to mean "warn on any self named mod files". The name should be plural also, that's my mistake.

Reply:
Yup that makes more sense